gix-config/fuzz/fuzz_targets/fuzz_file_section.rs

```diff
@@ -0,0 +1,42 @@
+#![no_main]
+
+use arbitrary::Arbitrary;
+use bstr::BStr;
+use gix_config::file::{Metadata, Section};
+use libfuzzer_sys::fuzz_target;
+use std::borrow::Cow;
+use std::hint::black_box;
+
+#[derive(Arbitrary, Debug)]
+struct Ctx<'a> {
+    name: Cow<'a, str>,
+    subsection: Option<&'a [u8]>,
+    meta: Metadata,
```

Comment:
I think if meta data wasn't generated, `arbitrary` can be avoided as feature in `gix-config`. It's OK to not generate everything, and if needed to mindfully iterate the input data with a set of common meta-data configurations.

Reply:
Easy, I'll just start with `Metadata::default()` and go from there.